resources/lang/sk/filters.php

```diff
@@ -9,5 +9,6 @@
     'sort_by' => [
         'chronologically' => 'chronologicky',
         'alphabetically' => 'abecedne',
+        'relevance' => 'relevancie',
```

Comment:
Would either of these sound better? (If so, it should be reflected in EN too)

```suggestion
        'relevance' => 'podľa relevancie',
```

```suggestion
        'relevance' => 'relevancia',
```